Ginger/Ginger/ConflictResolve/ConflictComparisonTreeViewItem.cs

```diff
@@ -38,11 +38,56 @@ public sealed class ConflictComparisonTreeViewItem : NewTreeViewItemBase, ITreeV
     {
         private readonly Comparison _comparison;
         private readonly Comparison.StateType[] _childrenStateFilter;
+        private readonly IDictionary<Comparison, IList<ConflictComparisonTreeViewItem>> _tviRepo;
 
-        public ConflictComparisonTreeViewItem(Comparison comparison, Comparison.StateType[] childrenStateFilter)
+        private TreeViewItem? _treeViewItem;
```

Comment:
Rename it to indicate its current treeview item

Reply:
done